### PR TITLE
Test topological sort

### PR DESCRIPTION
**Pull request by oddegen**
Changes proposed in this pull request:
- Edge cases tests for `pgr_topologicalSort`

@pgRouting/admins

### CODE REVIEW

File: pgtap/ordering/topologicalSort/edge_cases.pg [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+-- 1 vertex test
+
+PREPARE q3 AS
+SELECT id, source, 6 AS target, cost, -1 AS reverse_cost
+FROM edges
+WHERE id = 2;
+
+RETURN QUERY
+SELECT set_eq('q3', $$VALUES (2, 6, 6, -1, -1)$$, 'q3: Graph with one vertex');
```
Comment (cvvergara) [inline pgtap/ordering/topologicalSort/edge_cases.pg:61]: This test is irrelevant, you are testing what you are selecting in the query above.

**Comment by cvvergara** [comment]
Thanks we appreciate the effort

As the PR is focused on unit tests:

1) you have many irrelevant tests
2) comments of how the graph looks preferably in graphviz format
3) This looks more like working as: create a graph, run the code, use the results as the test
4) Missing tests like what happens when the graph is not DAG
5) Maybe find an internet example

**Remove the irrelevant tests for this PR to be merged**.

The noise from the locale directory will disappear (hopefully)

**Comment by oddegen** [comment]
@cvvergara I have removed the irrelevant tests
